### ISSUE TITLE
Expose command timing

### ISSUE THREAD

**Issue by bahmutov** [cypress-io/cypress#9263]
### What would you like?

I would like to know how long every command took to execute during the test to figure out how to speed things up. Was it the `cy.visit`? Or `cy.request`? Or `cy.get('selector').should('be.visible')`? 

We do have this information already - read https://www.cypress.io/blog/2020/05/22/where-does-the-test-spend-its-time/ and could provide it to the user either on demand, or by default

As the first step, I think we could simply add command timing to the DevTools Console output when clicking on the command

<img width="1127" alt="Screen Shot 2020-11-20 at 12 16 16 PM" src="https://user-images.githubusercontent.com/2212006/99829535-6818ab00-2b2a-11eb-95ba-026c094305d9.png">

We already report duration for some commands, like `cy.request`, but could report it for all of them.

Second step would show those durations for every command in the reporter, like this

<img width="1159" alt="Screen Shot 2020-11-20 at 12 23 23 PM" src="https://user-images.githubusercontent.com/2212006/99830107-3ce28b80-2b2b-11eb-89f6-1e2cbdd6cc18.png">

### Why is this needed?

People complain about Cypress being slow - https://twitter.com/sunpietro/status/1329545832262152203 is one example, and the first step to answering this question is understanding where the test is spending its time.

**Comment by ayshiff** [comment]
@bahmutov Can I work on this ? 👍

**Comment by bahmutov** [comment]
Sure, start with something simple like printing the command duration when clicked

**Comment by bahmutov** [comment]
For command log properties, we could use `log:changed` event https://on.cypress.io/catalog-of-events to add duration to every command

```js
Cypress.on('log:changed', options => {
  if (options.instrument === 'command' && options.consoleProps) {
    options.wallClockStoppedAt = Date.now()
    options.duration = +options.wallClockStoppedAt - (+ new Date(options.wallClockStartedAt))
    options.consoleProps.Duration = options.duration
  }
})
```


<img width="740" alt="Screen Shot 2020-12-04 at 9 36 48 AM" src="https://user-images.githubusercontent.com/2212006/101176270-3f0d1580-3614-11eb-95e4-675a674e671e.png">

**Comment by bahmutov** [comment]
Made my code into a tiny plugin https://github.com/bahmutov/cypress-timings and wrote a blog post https://glebbahmutov.com/blog/cypress-timings/

**Comment by ayshiff** [comment]
@bahmutov Do I have to close my Draft PR then ?

**Comment by bahmutov** [comment]
No, let it stay @ayshiff - no harm, and we are thinking about how to better grab those numbers and how to show them in the GUI

**Comment by AndreyYevtukhov** [comment]
Any updates on this? :)

**Comment by ghcheng85** [comment]
No updates?
